rules-reviewed/eap6/weblogic/weblogic.windup.xml

```diff
@@ -106,6 +106,22 @@
                     <message>Replace with the Java EE standard method `javax.transaction.TransactionManager.resume(Transaction tx)`.</message>
                     <tag>transactions</tag>
                     <tag>weblogic</tag>
+                    <quickfix name="Use java ee standard method Transaction Manager for resuming 1" type="REPLACE">
+                        <replacement>javax.transaction.TransactionManager.resume</replacement>
+                        <search>weblogic.transaction.TransactionManager.resume</search>
```

Comment:
right, I was switched to something else in my head and I forget on that it is pure string not AST ref.